src/RoverSelection/RoverSelection.scss

```diff
@@ -2,28 +2,100 @@
 @import "../Fonts.scss";
 
 
-.CuriosityRover{
-    font-size: x-small;
+.RoverSelection{
     background-color: $lightbackgroundcolor;
+    text-align: center;
+
+}
+@mixin roverstyle { 
+    width: 90%;
+    margin:auto;
+    text-align: center;
+    background-color:$darkbackgroundcolor;
+    border-radius: 25px;
+    margin-top: 3px;
+    padding: 3px;
+    padding-bottom: 15px;
+
+}
+@mixin iconstyle{
+    
+    border: 4px black;
+    background-color:$selectedbordercolor;
+    border-style:double;
+    border-radius: 50%;
+    
+}
+.imageicons{
```

Comment:
if you have a minute before merging it would be nice to have a space between these classNames and brackets just for formatting :)